example/hidden.js

```diff
@@ -20,6 +22,6 @@ app.model(User);
 var apiPath = '/api';
 explorer(app, { basePath: apiPath });
 app.use(apiPath, loopback.rest());
-console.log('Explorer mounted at localhost:' + port + '/explorer');
+console.log(g.f('Explorer mounted at {{localhost:%s/explorer}}', port));
```

Comment:
You have to run slt-globalize -e again for intl/en/messages.json to get updated with the brackets change.